src/FileConventions.Test/FileConventions.Test.fs

```diff
@@ -0,0 +1,45 @@
+module FileConventions.Test
+
+open System
+open System.IO
+open FileConventions
+open NUnit.Framework
+open NUnit.Framework.Constraints
+
+[<SetUp>]
+let Setup () =
+    ()
+
+[<Test>]
+let HasCorrectShebangTest1 () =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "DummyWithoutShebang.fsx")))
+    Assert.IsFalse(HasCorrectShebang(fileInfo))

```

Comment:
I prefer this assertion style: `Assert.That(HasCorrectShebang fileInfo, Is.EqualTo false)`

Reply:
Done